pkg/node/base.go

```diff
@@ -199,6 +200,25 @@ func getStagingPath(ctx context.Context, sp string, volID string) string {
 	return path.Join(sp, volID)
 }
 
+func getRemnantTargetMounts(ctx context.Context, target string, fs fs.Interface) ([]gofsutil.Info, bool, error) {
+	logFields := common.GetLogFields(ctx)
+	var targetMounts []gofsutil.Info
+	var found bool
+	mounts, err := getMounts(ctx, fs)
+	if err != nil {
+		log.Error("could not reliably determine existing mount status")
+		return targetMounts, false, status.Error(codes.Internal, "could not reliably determine existing mount status")
```

Comment:
Can we log the actual err as well?

Reply:
This matches the behavior of the getTargetMount() function. GetTargetMount() just returns the first mount, whereas the newly introduced method returns all the mounts.

Reply:
Thank you